### PR TITLE
[MBL-17635][Student][Teacher] "Add New" Button in Calendar Redsign a11y issues

### PR DESCRIPTION
**Pull request by tamaskozmer**
Test plan: See ticket. The switch access issue is not fixed, only the issues regarding TalkBack.

refs: MBL-17635
affects: Student, Teacher
release note: Fixed accessibility issues in the calendar.

## Checklist

- [x] Tested in dark mode
- [x] Tested in light mode
- [x] A11y checked

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

File: libs/pandautils/src/main/java/com/instructure/pandautils/compose/composables/ExpandableFloatingActionButton.kt [hunk 1]
```diff
@@ -91,14 +97,28 @@ fun ExpandableFloatingActionButton(
...
+
+        val announceRes = if (expanded.value) R.string.a11y_calendarActionsClosed else R.string.a11y_calendarActionsOpen
+        val onClickAnnounce = stringResource(announceRes)
+        val localView = LocalView.current
         FloatingActionButton(
             shape = shape,
             backgroundColor = backgroundColor,
             contentColor = contentColor,
-            onClick = { expanded.value = !expanded.value },
+            onClick = {},
```
Comment (hermannakos) [inline libs/pandautils/src/main/java/com/instructure/pandautils/compose/composables/ExpandableFloatingActionButton.kt:110]: You could remove this.

**Review by hermannakos — CHANGES_REQUESTED** [review]

**Comment by inst-danger** [comment]
<p>Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by hermannakos — APPROVED** [review]

**Comment by hermannakos** [comment]
QA + 1

**Comment by inst-danger** [comment]
<p>Parent Install Page</p>

**Comment by inst-danger** [comment]
<p>Teacher Install Page</p>

**Pull request merged**